rosco-manifests/src/main/java/com/netflix/spinnaker/rosco/manifests/BakeManifestRequest.java

```diff
@@ -15,6 +15,7 @@
   String outputName;
   List<Artifact> inputArtifacts;
   Map<String, Object> overrides;
+  String namespace;
```

Comment:
This is very specific to helm & k8s, I'm not sure this is the right spot. Is this the only way namespace can be configured?

Reply:
Is there anyway we could have a helm specific bake manifest request? Having this property exposed makes it easier to being existing Helm charts in because it's common practice to use things like `{{ .Release.Name }}` and `{{ .Release.Namespace }}`.

Reply:
Yes that sounds best -- @posox can you factor this into a separate helm bake manifest request class?

Reply:
@lwander @ethanfrogers I would prefer to use common property like `properties` with type `Map<String, Object>` or something else.
As I correct understand new request class requires new api call like `/api/v2/manifest/bake/helm`.

Reply:
We could have a new request class, or just pass the type as a parameter to the generic bake manifest endpoint

Reply:
@lwander added `properties` parameter to the generic request

Reply:
Awesome

> As I correct understand I should add new RequestMapping like /api/v2/manifest/bake/helm.

I think that's up to you -- we can either have a `type` field that decides how to cast the incoming request, or have different endpoints that orca decides call.

Reply:
hey @posox have you made any progress on refactoring this?

Reply:
@ethanfrogers yes, I'm working on refactoring.
@lwander I have a question. I don't understand how we can use `type` field in request for passing namespace.
As i correct understand it can be done using new field in request or using new endpoint for Helm bakery.

Reply:
@posox i think @lwander was saying use a `type` field to denote the type of BakeManifestRequest (Helm, other) and then cast the request to a concrete `HelmBakeManifestRequest`. That way we can support other types in the future and put implementation specific fields (`namespace` for Helm) on the implementation specific classes. Does that make sense?

Reply:
@ethanfrogers  how I can cast `BakeManifestRequest` to `HelmBakeManifestRequest`? In `BakeManifestRequest` the `namespace` field does not exists, so request cannot be casted correctly. I can change `RequestBody` to `HelmBakeManifestRequest`, but it looks strange.